flink-docs/src/main/java/org/apache/flink/docs/rest/RestAPIDocGenerator.java

```diff
@@ -131,7 +132,7 @@ public static void main(String[] args) throws IOException, ConfigurationExceptio
 
     @VisibleForTesting
     static void createHtmlFile(
-            DocumentingRestEndpoint restEndpoint, RuntimeRestAPIVersion apiVersion, Path outputFile)
+            DocumentingRestEndpoint restEndpoint, RestAPIVersion apiVersion, Path outputFile)
```

Comment:
I suggest move the generation of spec and doc of `RuntimeRestAPI` from `RestAPIDocGenerator` and `OpenApiSpecGenerator` called `RuntimeOpenApiSpecGenerator` and `RuntimeRestAPIDocGenerator` just like `SqlGatewayOpenApiSpecGenerator` and `SqlGatewayRestAPIDocGenerator` to make these two classes only keep some common logic.  And we can even rename `OpenApiSpecGenerator` to `RestAPISpecGenerator`.

Reply:
Done. `RuntimeOpenApiSpecGenerator` and `RuntimeRestAPIDocGenerator` have been added. And they will reuse the common logics from `RestAPIDocGenerator` and `OpenApiSpecGenerator`.